src/Avalonia.Input/DragEventArgs.cs

```diff
@@ -32,13 +35,25 @@ public Point GetPosition(IVisual relativeTo)
             return point;
         }
 
+        [Obsolete("Use constructor taking KeyModifiers")]
         public DragEventArgs(RoutedEvent<DragEventArgs> routedEvent, IDataObject data, Interactive target, Point targetLocation, InputModifiers modifiers)
             : base(routedEvent)
         {
-            this.Data = data;
-            this._target = target;
-            this._targetLocation = targetLocation;
-            this.Modifiers = modifiers;
+            Data = data;
+            _target = target;
+            _targetLocation = targetLocation;
+            Modifiers = modifiers;
+            KeyModifiers = (KeyModifiers)(((int)modifiers) & 0xF);
```

Comment:
Why not have `KeyModifiers = KeyModifiersUtils.ConvertToKey(modifiers);` like you have in RawDragEvent?

Reply:
Because it's for `RawInputModifiers`. I copied this from `KeyGesture`.